src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String report, String file) {
+        try {
+            Files.write(Path.of(file), report.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("There was an error writing report.", e);
```

Comment:
The exception message could be more informative by including the file path that caused the IOException to give more context to the error.